_includes/installation_location_table.html

```diff
@@ -0,0 +1,21 @@
+<table>
+  <thead>
+    <tr>
+      <td></td>
+      <td>{{ t.pagecontent.intel }}</td>
+      <td>{{ t.pagecontent.arm }}</td>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>{{ t.pagecontent.macos }}</td>
+      <td><code>/usr/local</code></td>
+      <td><code>/opt/homebrew</code></td>
+    </tr>
+    <tr>
+      <td>{{ t.pagecontent.linux}}</td>
+      <td><code>/home/linuxbrew</code></td>
+      <td><code>/home/linuxbrew</code></td>
```

Comment:
```suggestion
      <td><code>/home/linuxbrew/.linuxbrew</code></td>
      <td><code>-</code></td>
```
There isn't really a default prefix on ARM Linux, since the installer won't run on it, I think. (Might be nicer if the `-` were centred; could also be an `N/A`.)

Reply:
Agreed. I don't think it's worth a 2x2 table for that reason

Reply:
I preferred the table to a list in a mockup I did first. 

What is the likelihood that we'd add more rows or columns in the table?  Probably nil.

Given these:

* It's pretty safe to say that macOS isn't coming to a new architecture anytime soon.
* We'd probably always use `/home/linuxbrew/.linuxbrew` on _any_ Linux, or change it on _all_ of them; there's not reason for an arch-specific installation path on Linux
* Any future OS that we'd support would likely have an installation prefix treated like Linux's, e.g. `/home/bsdbrew/.bsdbrew`


...I can safely predict that it will not need to expand anytime soon.


Could we use `∄` instead of a `-`? `&#8708;` would do it. "N/A" would probably have to be localized. I'll take the `-` if you all don't think enough people would know what ∄ means ("there does not exist").

Reply:
Personally I still think linking out to something on docs.brew.sh is more appropriate here. This homepage is just an example, it's not worth enumerating information here (we don't do it for anything else really)